src/compass/s1_geo2rdr.py

```diff
@@ -87,17 +87,16 @@ def run(cfg: dict):
         # Execute geo2rdr
         geo2rdr_obj.geo2rdr(topo_raster, burst_output_path)
 
-    dt = str(timedelta(seconds=time.time() - t_start))
-    info_channel.log(f"geo2rdr successfully ran in {dt} (hr:min:sec)")
+    dt = str(timedelta(seconds=time.time() - t_start)).split(".")[0]
+    info_channel.log(f"s1_geo2rdr burst successfully ran in {dt} (hr:min:sec)")
 
 
 if __name__ == "__main__":
     """Run geo2rdr from command line"""
-    geo2rdr_parser = YamlArgparse()
+    parser = YamlArgparse()
 
     # Get a runconfig dict from command line arguments
-    geo2rdr_runconfig = RunConfig.load_from_yaml(
-        geo2rdr_parser.args.run_config_path, 'geo2rdr')
+    cfg = RunConfig.load_from_yaml(parser.args.run_config_path, 's1_geo2rdr')
```

Comment:
I don't quite understand this part. Should this line be as below for `s1_geo2rdr`, `s1_rdr2geo` and `s1_resample`?

```
cfg = RunConfig.load_from_yaml(parser.args.run_config_path, 's1_cslc_radar')
```

Reply:
> I don't quite understand this part. @vbrancat @LiangJYu Should this line be as below for `s1_geo2rdr`, `s1_rdr2geo` and `s1_resample`?
> 
> ```
> cfg = RunConfig.load_from_yaml(parser.args.run_config_path, 's1_cslc_radar')
> ```

[`load_from_yaml`](https://github.com/yunjunz/COMPASS/blob/70633bcf290e69339d789cbb910092efeb9fb777/src/compass/utils/runconfig.py#L310) passes the 2nd parameter to [`load_validate_yaml`](https://github.com/yunjunz/COMPASS/blob/70633bcf290e69339d789cbb910092efeb9fb777/src/compass/utils/runconfig.py#L35). In `load_validate_yaml`, the 2nd parameter/`workflow_name` is used to determine `schema_name`.

Reply:
Thank you @LiangJYu, I will change the arg input for s1_geo2rdr/rdr2geo/resample then.